bgpd/bgpd.c

```diff
@@ -4763,40 +4767,42 @@ struct peer_flag_action {
 };
 
 static const struct peer_flag_action peer_flag_action_list[] = {
-	{PEER_FLAG_PASSIVE, 0, peer_change_reset},
```

Comment:
Please undo this formatting, it's not related.

Reply:
Fixed

Reply:
If this format is not modified, it will generate a clang warning. I'm not sure how to fix it. If I use the clang-format tool to fix it, it will revert to the previous form. Please help me figure out how to handle this.

Reply:
Just ignore the warning the code you didn't touch.